tox.ini

```diff
@@ -188,12 +217,19 @@ ignore_missing_imports = True
 
 description = generate coverage report
 
-basepython = python
+depends = test-py{36,37,38,39,py3}
```

Comment:
This makes parallel `tox` builds work, as it wait for other environments before running `coverage`.

Reply:
Oh nice, I've been meaning to do this.